pkg/utils/github.go

```diff
@@ -218,3 +223,32 @@ func generateIndexYAML(repoRoot string) (*repo.IndexFile, map[string]string, err
 
 	return indexFile, resourceDirs, nil
 }
+
+// ParseKubeResoures parses a YAML content and returns kube resources in byte array from the file
+func ParseKubeResoures(file []byte) [][]byte {
```

Comment:
https://github.com/kubernetes/apimachinery/blob/master/pkg/util/yaml/decoder.go ?

Reply:
Thanks for reviewing. The code I wrote suites our purpose better.